# vim: set ft=rb:

class @
  def @main()``
    try
      throw("Error")
    catch(msg : String)
      trace("Error occurred: #{msg}")
    end
  end
end
